src/Orleans.Streaming/QueueId.cs

```diff
@@ -42,68 +39,52 @@ private QueueId(string queuePrefix, uint id, uint hash)
         /// <param name="queueId">The queue identifier.</param>
         /// <param name="hash">The hash.</param>
         /// <returns>The queue identifier.</returns>
-        public static QueueId GetQueueId(string queueName, uint queueId, uint hash)
-        {
-            var key = new QueueId(queueName, queueId, hash);
-            return queueIdInternCache.Intern(key, key);
-        }
+        public static QueueId GetQueueId(string queueName, uint queueId, uint hash) => new(queueName, queueId, hash);
```

Comment:
We should throw if `queueName` is null, given we're using it for logic (`IsDefault`)